lib/ae_mdw/db/origin.ex

```diff
@@ -143,21 +143,12 @@ defmodule AeMdw.Db.Origin do
   end
 
   defp hc_contracts do
-    with {:ok, {hardfork_key, _hf_val}} <-
-           :aeu_env.find_config(["chain", "hard_forks"], [
-             :user_config,
-             :schema_default,
-             {:value, {@iris_protocol, 0}}
-           ]),
-         {protocol, ""} <- Integer.parse(hardfork_key),
-         {:ok, hc_contracts_json} <-
-           :aec_fork_block_settings.hc_seed_contracts(protocol, :aec_governance.get_network_id()) do
```

Comment:
During localnet tests there was a mapping from "5" to a custom protocol id like:
```
    "hard_forks": {
      "5": 0
    }
```
If the Node team won't use the mapping anymore it makes sense to pass 5 directly instead of looking to which protocol the hard fork is associated.

Reply:
It still uses that same format, but didn't return a tuple , it instead returned a map:
```elixir
iris_protocol = "5"
:aeu_env.find_config(["chain", "hard_forks"], [
  :user_config,
  :schema_default,
  {:value, {iris_protocol, 0}}
])
{:ok, %{"5" => 0}}
```
which ended up matching with `_error_missing` and not returning the contracts

Reply:
The only remaining question would be if we want to remove the mapping and use always the iris protocol (no more hardkfork mapping for dev).